app/views/seriesDetails.scala.html

```diff
@@ -10,7 +10,7 @@
 
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-        <a href="@routes.DashboardController.dashboard()" class="govuk-back-link">Back</a>
+@*        <a href="@routes.DashboardController.dashboard()" class="govuk-back-link">Back</a>*@
```

Comment:
Could you delete these rather than commenting out? Commented-out code is confusing to work with, because if you aren't aware of the context then you don't know if it was supposed to be a temporary comment that's been committed by accident or if it was deliberate.

They'll be in the git history when we want to bring them back. 🙂

Reply:
Yes of course! Sorry about that.